packages/wonder-blocks-form/src/components/__docs__/radio.stories.js

```diff
@@ -0,0 +1,160 @@
+
+export default {
+    title: "Form / Radio",
+    component: Radio,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+    },
+};
+
+export const Default: StoryComponentType = (args) => <Radio {...args} />;
+
+Default.args = {
+    checked: false,
+    onChange: () => {},
+};
+
+Default.parameters = {
+    chromatic: {
+        // We already have screenshots of another story that covers
+        // this and more cases.
+        disableSnapshot: true,
+    },
+};
+
+export const Controlled: StoryComponentType = () => {
+    const [checked, setChecked] = React.useState(false);
+    return <Radio checked={checked} onChange={setChecked} />;
+};
+
+Controlled.parameters = {
+    chromatic: {
+        // Disabling because this doesn't test visuals, it tests
+        // that the `checked` state works as expected.
+        disableSnapshot: true,
+    },
+    docs: {
+        storyDescription: `Use state to keep track of whether
+        the radio button has been checked. A radio button cannot be unchecked
+        by the user once it has been checked. It would become unchecked if a
+        different radio button is selected as part of a radio group.`,
+    },
+};
+
+export const Variants: StoryComponentType = () => (
+    <View style={styles.row}>
+        <Radio checked={false} style={styles.marginRight} onChange={() => {}} />
+        <Radio checked={true} style={styles.marginRight} onChange={() => {}} />
+        <Radio
+            error={true}
+            checked={false}
+            style={styles.marginRight}
+            onChange={() => {}}
+        />
+        <Radio
+            error={true}
+            checked={true}
+            style={styles.marginRight}
+            onChange={() => {}}
+        />
+        <Radio
+            disabled={true}
+            checked={false}
+            style={styles.marginRight}
+            onChange={() => {}}
+        />
+        <Radio disabled={true} checked={true} onChange={() => {}} />
+    </View>
+);
+
+Variants.parameters = {
+    docs: {
+        storyDescription: `The radio button has various styles for
+        clickable states. Here are sets of default radio buttons,
+        radio buttons in an error state, and disabled radio buttons.`,
+    },
+};
+
+export const WithLabel: StoryComponentType = () => {
+    const [checked, setChecked] = React.useState(false);
+
+    return (
+        <Radio
+            label="Easy"
+            description="Opt for a less difficult exercise set."
+            checked={checked}
+            onChange={setChecked}
+        />
+    );
+};
+
+WithLabel.parameters = {
+    docs: {
+        storyDescription: `The radio button can have an optional label
+            and description. This allows it to be used as a settings-like item,
+            as opposed to its usage in a radio grid.`,
+    },
+};
+
+export const AdditionalClickTarget: StoryComponentType = () => {
```

Comment:
This is also a problem with Checkbox, not just with Radio

Reply:
ahh yeah, I didn't notice the repro steps at first and that's super helpful.

Did some debugging and found out that if you are in the `Docs` tab and point to a specific story, then it will include the `Canvas` version in a separate wrapper, but using `hidden=true`. The issue there is that the DOM ends up with two nodes with the same ID, so the `labelFor` cannot recognize properly which `input` is referenced to that `label`.

<img width="542" alt="Screen Shot 2022-06-07 at 9 10 36 AM" src="https://user-images.githubusercontent.com/843075/172388888-6e224af4-1a08-43f6-8d23-ae6bfe9d5c33.png">

In summary, it's definitively a Storybook issue, so don't worry about it. One thing that we could do is to file a ticket on the SB repo to see if they get that fixed at some point, but nothing else to do as part of this PR.